src/modules/SourceDialogs.rb

```diff
@@ -2013,6 +2015,14 @@ def scc_repos_widget
       display_scc ? Left(RadioButton(Id(:sccrepos), _(WIDGET_LABELS[:sccrepos]))) : Empty()
     end
 
+    def network_button
+      if Mode.installation || Mode.live_installation || Mode.update
```

Comment:
JFYI, `Mode.installation` is `true` for modes: installation, autoinstallation, live_installation